src/frontend/src/instance/distributed.rs

```diff
@@ -556,6 +550,11 @@ impl DistInstance {
             .await
             .context(error::RequestMetaSnafu)?;
 
+        // Invalidates local cache ASAP.
+        self.catalog_manager()
+            .invalidate_table(catalog_name, schema_name, table_name, table_id)
+            .await;
+
```

Comment:
submit ddl -> invalid cache -> get table and put into cache -> submit ddl update metadata

What would happen if the above execution process was carried out?